hnn_core/params.py

```diff
@@ -323,9 +322,9 @@ def create_pext(p, tstop):
     }
 
     # ensures time interval makes sense
-    p_ext = _feed_validate(p_ext, feed_prox, tstop)
+    p_common = _feed_validate(p_common, feed_prox, tstop)
```

Comment:
```suggestion
    p_prox = _feed_validate(p_prox, feed_prox, tstop)
```

? I'm a bit confused btw this common and prox thing

Reply:
-1 the function `_feed_validate` appends to the list `p_common` so that both distal and proximal "common" inputs end up in the same parameter _list_. Note that `p_unique` in the same function (`create_pext`) has always been a `dict`... See also comment above regarding the 'commonness' of what we all thought were somehow specific to 'rhythmic'.